csi/proto/mayastornodeplugin.proto

```diff
@@ -0,0 +1,44 @@
+// The definition of mayastor node plugin gRPC interface.
+// The node plugin service runs on all nodes running
+// the Mayastor CSI node plugin, and is complementary
+// to the CSI node plugin service.
+
+// This interface is supposed to be independent on particular computing
+// environment (i.e. kubernetes).
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "io.grpc.examples.mayastornodeplugin";
+option java_outer_classname = "MayastorNodePluginProto";
+
+package mayastornodeplugin;
+
+// Service for freezing and unfreezing file systems.
+service MayastorNodePlugin {
+  // Freeze the file system identified by the volume ID
+  // no check is made if the file system had been previously frozen.
+  rpc FreezeFS (FreezeFSRequest) returns (FreezeFSReply) {}
+  // Unfreeze the file system identified by the volume ID,
+  // no check is made if the file system had been previously frozen.
+  rpc UnfreezeFS (UnfreezeFSRequest) returns (UnfreezeFSReply) {}
+}
+
+// The request message containing ID of the volume to be frozen
+message FreezeFSRequest {
+  string volume_id = 1;
+}
+
+// The response message for the freeze request.
+message FreezeFSReply {
+}
```

Comment:
Might be better to create an explicit Null reply message as we do for the mayastor API (see Mayastor/rpc/proto/mayastor.proto)